### ISSUE TITLE
Bug/Feature ( Need to custom paths to be ignored for the rule "import/no-unresolved" ) in eslint config

### ISSUE THREAD

**Issue by shivashanmugam** [eslint/eslint#18782]
### Environment

Node version: v18.12.1 
npm version: 
Local ESLint version: ^8.11.0
Global ESLint version:  8.19.2
Operating System: mac OS ( 14.6 (23G80) )
----

### What parser are you using?

@typescript-eslint/parser

### What did you do?

I am currently using module federation in my UI application, The below is the webpack configuration
through which I can use library called `host` 

```js
 new ModuleFederationPlugin({
      name: "oms",
      filename: "oms_remoteEntry.js",
      remotes: {
        host: "host@http://localhost:3000/host_remoteEntry.js", 
      },
}),
```
----

I am using the `host` library in my app like below
.
.
`import useGlobalStore from "host/GlobalStore";`
.
.
<img width="1480" alt="image" src="https://github.com/user-attachments/assets/876b3834-8142-4142-baf2-2d8ed6aad60d">
.
.

But that creates Eslint error 
`Unable to resolve path to module 'host/GlobalStore'.eslint[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.29.1/docs/rules/no-unresolved.md)`

----
I want to resolve the issue without adding  in my `eslint.rc`
`"import/no-unresolved": "off",`

I have also added `tsremote_declaration.ts`
`declare module 'host/GlobalStore';`

### What did you expect to happen?

Basically I want eslint to ignore the unresolved path which starts has `host` as prefix

### What actually happened?

<img width="1480" alt="image" src="https://github.com/user-attachments/assets/876b3834-8142-4142-baf2-2d8ed6aad60d">

### Link to Minimal Reproducible Example

https://stackoverflow.com/questions/77435417/how-to-resolve-module-federation-import-paths-in-eslint-with-typescript

### Participation

- [X] I am willing to submit a pull request for this issue.

### Additional comments

I have already found a question related to this in stackoverflow, But did't get right solution
.
https://stackoverflow.com/questions/77435417/how-to-resolve-module-federation-import-paths-in-eslint-with-typescript

**Comment by aladdin-add** [comment]
Thanks for the issue. the rule `import/no-unresolved` was from a 3rd-party plugin, it's not maintained by the eslint team. You'll need to ask help in its repo: https://github.com/import-js/eslint-plugin-import
